pom.xml

```diff
@@ -171,27 +172,27 @@
 				</plugin>
 				<plugin>
 					<artifactId>maven-dependency-plugin</artifactId>
-					<version>3.0.2</version>
+					<version>3.1.1</version>
 				</plugin>
 				<plugin>
 					<artifactId>maven-jar-plugin</artifactId>
 					<!-- warning: before upgrading to a newer version of this plugin
 					      there are some config changes required.
 					      see https://github.com/biojava/biojava/issues/531
 					      -->
-					<version>3.0.2</version>
+					<version>3.1.0</version>
```

Comment:
Could you remove the comment above this line?

Reply:
Good catch, I'll push a commit